<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMeasurment</class>
 <widget class="QWidget" name="StartMeasurment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Параметры измерения</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_probe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string notr="true">Режим измерния</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton_single">
        <property name="text">
         <string>Одинарные пробы</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_double">
        <property name="text">
         <string>Парные пробы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_ch">
       <property name="title">
        <string>Выбор каналов</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_ch1">
          <property name="text">
           <string>Канал 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ch2">
          <property name="text">
           <string>Канал 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ch3">
          <property name="text">
           <string>Канал 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ch4">
          <property name="text">
           <string>Канал 4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>№ проб</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_ch1">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ch2">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ch3">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ch4">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_t">
       <property name="text">
        <string>Время записи, сек</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_time</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_time">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_incube">
       <property name="text">
        <string>Время инкубации</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_incube">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>180</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_incube_2">
       <property name="text">
        <string>Время инкубации 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_incube_2">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>180</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_next">
       <property name="text">
        <string>Продолжить</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_ch1</tabstop>
  <tabstop>lineEdit_ch1</tabstop>
  <tabstop>checkBox_ch2</tabstop>
  <tabstop>lineEdit_ch2</tabstop>
  <tabstop>checkBox_ch3</tabstop>
  <tabstop>lineEdit_ch3</tabstop>
  <tabstop>checkBox_ch4</tabstop>
  <tabstop>lineEdit_ch4</tabstop>
  <tabstop>radioButton_double</tabstop>
  <tabstop>lineEdit_time</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
